src/routes/nexus.ts

```diff
@@ -0,0 +1,7 @@
+import express from 'express';
+const router: express.Router = express.Router();
+import { nexusController } from '../controllers/nexusController';
+
+router.get('/', nexusController.nexusIndex);
```

Comment:
Why not combine nexusController and this file? (There may be a good reason, I'm just curious.)

Reply:
I was following the [design pattern of isolating routes from controllers here](https://developer.mozilla.org/en-US/docs/Learn/Server-side/Express_Nodejs/routes). The idea is that by isolating the controllers, you can have route files forward requests at different endpoints to different controller files, making things a bit cleaner and more organized. In this case right now there is no particular reason for that isolation, but down the line it allows us to more cleanly add more controllers. A possible use case might be adding social integration to the nexus page with its own controller to parse a Twitter feed or something.